src/Core/Managed/Shared/Extensibility/TelemetryConfiguration.cs

```diff
@@ -133,6 +134,15 @@ public IList<ITelemetryInitializer> TelemetryInitializers
         }
 
         /// <summary>
+        /// Gets the list of <see cref="IMetricProcessor"/> objects used for custom metric data processing
+        /// before client-side metric aggregation process.
+        /// </summary>
+        public IList<IMetricProcessor> MetricProcessors
```

Comment:
Name `processor` makes it inconsistent with the `TelemetryProcessors` which combines the chain rather than sequential processing. Do you have any other names in mind?